forc-pkg/src/manifest.rs

```diff
@@ -223,6 +223,8 @@ pub struct BuildProfile {
     pub json_abi_with_callpaths: bool,
     #[serde(default)]
     pub error_on_warnings: bool,
+    #[serde(default)]
```

Comment:
I don't think you need to have the `#[serde(default)]` annotation here. Here's the doc that explains what it does, though I think it was just leftover from the field you had removed here.

Reply:
Just got that done and committed, thank you for the patience :)